src/db/config.ts

```diff
@@ -3,30 +3,42 @@ import dotenv from "dotenv"
 
 dotenv.config()
 
-const sequelize =
-  process.env.NODE_ENV === "test"
-      ? new Sequelize(
-          "postgresql://postgres:reaYBl4M1hbiDdxjnTdr@containers-us-west-194.railway.app:7407/railway", {
-              logging: false,
-              sync: {
-                  force: false,
-                  alter: { drop: false },
-              },
-          }
-      )
-      : new Sequelize({
-          dialect: "postgres",
-          host: String(process.env.DB_HOST),
-          port: Number(process.env.DB_PORT),
-          database: String(process.env.DB_NAME),
-          username: String(process.env.DB_USER),
-          password: String(process.env.DB_PASSWORD),
-          logging: false,
-          sync: {
-              force: false,
-              alter: { drop: false },
-          },
-      })
+// const sequelize =
```

Comment:
Please uncomment these changes to enable the automatic switch between dev and test environmenrs